var/spack/repos/builtin/packages/conduit/package.py

```diff
@@ -122,6 +122,10 @@ class Conduit(CMakePackage):
     depends_on("hdf5@1.8.19:1.8~shared~cxx", when="+hdf5+hdf5_compat~shared")
     depends_on("hdf5~cxx", when="+hdf5~hdf5_compat+shared")
     depends_on("hdf5~shared~cxx", when="+hdf5~hdf5_compat~shared")
+
+    # conduit uses a <1.10 api version.
```

Comment:
The `depends_on` below suggests:
```suggestion
    # conduit uses a <=1.10 api version.
```